implementation/parallel-to-sns-scatter-gather/README.md

```diff
@@ -1,21 +1,25 @@
 
-This project is the CDK implementation of ['Replace Parallel state with SNS for Scatter-Gather'](https://serverlessland.com/refactoring-serverless/parallel-to-sns-scatter-gather) pattern. It shows how yo can use SNS in CDK to replace AWS Step Functions `Map` state to implement the [Scatter-Gather](https://www.enterpriseintegrationpatterns.com/patterns/messaging/BroadcastAggregate.html) pattern.
+This project is the CDK implementation of ['Replace Parallel state with SNS for Scatter-Gather'](https://serverlessland.com/refactoring-serverless/parallel-to-sns-scatter-gather) refactoring. It shows how you can use Amazon SNS plus an Aggregator function instead of an Amazon Step Functions to implement the [Scatter-Gather](https://www.enterpriseintegrationpatterns.com/patterns/messaging/BroadcastAggregate.html) integration pattern.
+
+See [Replace Parallel with SNS Scatter-Gather](/patterns/parallel_to_sns_scatter_gather.md) for details on the refactoring from a conceptual point of view.
 
 ## How it works
 
-The code will deploy two stacks:
+As example use case we use a Car Rental search engine(```requester```) that sends a quote request to several Car Rentals(```responder```). The Car Rental search engine returns the aggregated(```aggregator```) results from multiple suppliers to the user.
+
+The code deploys two stacks:
 
 1. ScatterGatherWithParallelStack
 2. ScatterGatherWithSNSStack
 
-Both stacks deploy 3 lambda functions (application code is re-used for both stacks), ```requester```, ```responder``` and ```aggregator```. The ```requester``` sends a request for quote message providing it with a ```uuid```, multiple ```responder``` will processes the message and outputs a modified message containing a ```price_quote``` specific to the vendor (provided as config parameter in ```cdk.json``` file). The ```aggregator``` aggregates the received quotes.
+Both stacks deploy 3 lambda functions (application code is re-used for both stacks), ```requester```, ```responder``` and ```aggregator```. The ```requester``` sends a request for quote message with an ```uuid```. Multiple ```responder```(Car Rental) process the message modifying the message adding a ```price_quote```. The Car Rentals configuration parameters are defined in the ```cdk.json``` file. Ultimately, the ```aggregator``` function aggregates the received quotes.
 
-ScatterGatherWithParallelStack deploys a Step Function workflow using the parallel state to send a quote request 2 ```responder```. Each ```responder``` adds a price_quote to the original message. When all parallel executions have completed the result is aggregated by the Step Function parallel state.
+ScatterGatherWithParallelStack deploys a Step Function workflow using the parallel state to send a quote request to ```responder```. Each ```responder``` adds a price_quote to the original message. When all parallel executions have completed the result is aggregated by the Step Function parallel state.
```

Comment:
Minor: is the fact that the responder modifies the original message important (i.e. part of the interface contract)? Else I would just state that the responder returns a price quote.
Minor: I would say "The Step Functions parallel state aggregates the results from all responders" (plural and active)